frontend/src/components/details/NavbarComponent.js

```diff
@@ -12,40 +13,55 @@ import { Link } from 'react-router-dom';
 const NavbarComponent = (props) => {
     if (!props.loadComparePage) {
         return (
-            <div className='navbar w3-row' vertical='center' horizontal='space-between'>
+            <div className='navbar' vertical='center' horizontal='space-between'>
```

Comment:
I don't know whether vertical center and horizontal space between work inside of a div element as well. I personally don't think so. In that case we can remove them.